docs/demo/buttons.html

```diff
@@ -4,71 +4,71 @@
 ---
 
 <div class="body-wrapper">
-    
+
     <h1 class="page-header">Buttons</h1>
-    
+
     <h4 class="sub-heading">Button styles</h4>
-    
+
     <div class="message message-info">
         <h4><i class="icon fw fw-info"></i>Bootstrap</h4>
         <p>
-            For additional details please check bootstrap documentation: 
+            For additional details please check bootstrap documentation:
             <a href="http://getbootstrap.com/css/#buttons" target="_blank">http://getbootstrap.com/css/#buttons</a>
         </p>
     </div>
-    
+
     <div class="code">
         <div data-lang="html">
 
-    <!-- buttons -->
-    <button type="button" class="btn btn-default">Default</button>
-    <button type="button" class="btn btn-primary">Primary</button>
-    <button type="button" class="btn btn-secondary">Secondary</button>
-    <button type="button" class="btn btn-success">Success</button>
-    <button type="button" class="btn btn-info">Info</button>
-    <button type="button" class="btn btn-warning">Warning</button>
-    <button type="button" class="btn btn-danger">Danger</button>
-    <button type="button" class="btn btn-link">Link</button>
-    <!-- /buttons -->
+            <!-- buttons -->
```

Comment:
can we affix or suffix some string to uniquely identify "btn-default" id from "btn-default" class, there shouldnt be an ambiguity.